libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Utils.java

```diff
@@ -67,7 +67,8 @@
     public static final String FORMAT_DATE = "yyyy-MM-dd";
     public static final String FORMAT_DATE_TIME = "yyyy-MM-dd HH:mm:ss";
     public static final Comparator<ReleaseLink> RELEASE_LINK_COMPARATOR = Comparator.comparing(rl -> getReleaseFullname(rl.getVendor(), rl.getName(), rl.getVersion()).toLowerCase());
-
+    public static final Predicate<Attachment> isApprovedCLI = attachment -> attachment.getCheckStatus().equals(CheckStatus.ACCEPTED)
```

Comment:
Please move this predicate to method level.

Reply:
Done.